README.md

```diff
@@ -18,3 +18,25 @@ This integration has been tested with Solarfocus eco<sup>manager-touch</sup> ver
 | Biomassboiler (_Kessel_) | :white_check_mark: | 
 
 _Note: The number of supported Heating Circuits, Buffers, and Boilers could be extended in the future_
+
+## Usage
+
+```python
+from pymodbus.client.sync import ModbusTcpClient as ModbusClient
```

Comment:
```suggestion
from pymodbus.client import ModbusTcpClient as ModbusClient
```
I'm confused why, but for me the `.sync` path is not available anymore after switching to Python 3.10 and re-installing _pymodbus_.  So not sure, if this is just for me, or a general thing.

Reply:
I think they changed the namespace with version 3.0.0 if we want to remove the .sync we should bump the minimum version to ^3.0.0.